sections/introduction/introduction-chapter.tex

```diff
@@ -1,20 +1,21 @@
 \noindent
 \section{Introduction}
 \chapter{Recommender systems (RS)}\label{ch:introduction}
-have become an important part of everyday life on various online platforms \cite{youtuberecommendation, industryperspective}.\\\
+have become an important part of everyday life on various online platforms.
+\\
 With the tremendous amount of information available to users, finding what you need without help can be overwhelming.
-One way RS have attempted to aid users is through collaborative filtering (CF), where the preferences and similarities between users and items are used to generate recommendations. 
+One way attempted to aid users with RS is through collaborative filtering (CF), where the preferences and similarities between users and items are used to generate recommendations. 
 Graph Convolutional Network (GCN)-based models have gained high popularity and also achieved great results \cite{NGCF,LightGCN,KGAT}, due especially to their ability to aggregate information from neighboring nodes.\\
-However, most of the GCN models focus purely on user-item interactions and do not take into account additional information that will usually be available in both existing datasets and real life.
+However, most of the GCN-based models for RS focus purely on user-item interactions and do not take into account additional information that will usually be available in both existing datasets and real life.
 This additional information could be information from a user profile, including their age and location, or information about the items such as user-specified tags for attractions, or the genres of a movie.
 There is usually also some contextual information available concerning the interaction taking place, which may be something as simple as a timestamp, or information about the current emotional state of the user.\\
-This side-information and contextual information can be beneficial for prediction performance in some situations \cite{ContextImportance, ContextImportance2, ContextImportance3}, as well as assist in alleviating certain issues such as data sparsity and cold starts \cite{SideInfoDefinition}.
-Using this context information is a sub-genre of recommender systems called context-aware recommender systems (CARS) \cite{carsprogress}, where the associated context of an interaction is examined when creating a prediction.
+This side-information and contextual information can be beneficial for prediction performance \cite{ContextImportance2}, as well as assist in alleviating certain issues such as data sparsity and cold starts \cite{SideInfoDefinition}.
```

Comment:
hvorfor fjerne vi kilder her?

Reply:
Vi synes der var for mange kilder at skulle læse igennem, og blev enige om at de sagde lidt det samme, men kan godt se hvad du mener med at det er smart at have flere kilder til at bakke en vigtig påstand op